packages/frontend/src/components/connect-wallet/index.tsx

```diff
@@ -80,41 +66,25 @@ export const ConnectWallet = ({
         setConnectPopoverOpen(false);
     }, []);
 
-    const handleAccountPopoverOpen = useCallback(() => {
-        setAccountPopoverOpen(true);
+    const handleSettingsDrawerOpen = useCallback(() => {
+        setSettingsDrawerOpen(true);
     }, []);
 
-    const handleAccountPopoverClose = useCallback(() => {
-        setAccountPopoverOpen(false);
+    const handleSettingsDrawerClose = useCallback(() => {
+        setSettingsDrawerOpen(false);
     }, []);
 
-    const handleNetworkSwitchClick = useCallback(
-        async (chainId: number) => {
-            try {
-                await activeConnector?.switchChain?.(chainId);
-            } catch (error) {
-                console.warn("could not switch network", error);
-            }
-            setNetworksPopoverOpen(false);
-        },
-        [activeConnector],
-    );
-
-    const multipleEnabledChains = Object.keys(ENABLED_CHAINS).length > 1;
-    const chainId = chain?.id || Number.MAX_SAFE_INTEGER;
-    const chainName = chain?.name || t("connect.wallet.unknown");
-    const supportedChain = !!chainId && !!ENABLED_CHAINS[chainId];
-    const Logo = supportedChain
-        ? ENABLED_CHAINS[chainId as ChainId].logo
-        : Error;
     return (
         <>
-            {!__LIBRARY_MODE__ && (
-                <NetworksPopover
-                    open={networksPopoverOpen}
-                    anchor={networksPopoverAnchor}
-                    onNetworkSwitch={handleNetworkSwitchClick}
-                    ref={networksPopoverRef}
+            {width >= 768 ? (
```

Comment:
Can we transform this into the usage of media queries through Tailwind, which should be more performant?

Reply:
The issue here is that with media queries both elements (mobile and desktop) would be attached to the DOM (one with `display: none`), and then both the `useClickAway` would be invoked in either desktop or mobile, causing the drawer to close at any click.

Reply:
Can in some way `pointer-events: none` help here?